setup.py

```diff
@@ -10,9 +11,18 @@ def read(*parts):
         return f.read()
 
 
+def find_version(*file_paths):
```

Comment:
I don't have strong feelings about where to put version info so am open to other systems. I like this way mostly because it seems reasonably simple.

Reply:
Fine with me.